addons/sale/tests/test_sale_order_down_payment.py

```diff
@@ -434,11 +434,13 @@ def test_warning_on_invoice_with_credit_limit(self):
 
         # Check that partner_a's credit is 0.0.
```

Comment:
Please update the comment to reflect the changes you made :slightly_smiling_face: